### DISCUSSION TITLE
Make the order of biomes in PVP settings make more sense

### DISCUSSION THREAD

**Discussion by 4-S-H** [FakeFishGames/Barotrauma#14941] [Ideas and suggestions]
Currently the biomes are listed in a strange order. Would be better if they are sorted in their order in a campaign.
![image](https://github.com/user-attachments/assets/498a2550-1de4-408e-8f44-ef4a3ab2fca8)

**Comment by Regalis11** [comment]
Addressed in https://github.com/FakeFishGames/Barotrauma-development/commit/0b29e67ac014afd9b3988b1c2e999f91ff6c2b25 (commit in our private repo), fix coming up in a future update!
